test/resolver.spec.ts

```diff
@@ -1,7 +1,84 @@
-import { AsyncAPIDocumentV2 } from '../src/models';
+import { AsyncAPIDocumentV2, AsyncAPIDocumentV3 } from '../src/models';
 import { Parser } from '../src/parser';
 
 describe('custom resolver', function() {
+  it('Reply channel + messages[] Ids come from JSON Pointer', async function() {
+    const parser = new Parser();
+
+    const documentRaw = `{
```

Comment:
This doc is the one @AceTheCreator shared in https://github.com/asyncapi/parser-js/issues/873.
We could remove it or adapt it, but we need something to test this anyway.